doc/docs/master/how-tos/use-jupyterhub/index.md

```diff
@@ -0,0 +1,75 @@
+# Use JupyterHub with `python-pachyderm`
+
+This section describes how you can use the `python-pachyderm`
+client from within the JupyterHub UI.
+
+!!! note
+    You need to have Pachyderm and JupyterHub installed on the
+    same Kubernetes cluster as described in
+    [Deploy Pachyderm with JupyterHub](../../deploy-manage/deploy/deploy-pachyderm-jupyterhub.md).
+    Currently, an existing JupyterHub deployment cannot be
+    integrated with Pachyderm.
```

Comment:
This isn't true, you can modify an existing JupyterHub deployment, and we give some (very incomplete) guides on how to do so in the jupyterhub-pachyderm repo. You just can't use `init.py`, so the steps required are quite different.

Reply:
I'm just going to remove the last sentence but maybe let's address the existing JH integration on a case-by-case basis until we have a proper process?